src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -80,6 +82,41 @@ public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stu
         return $filter & Caster::EXCLUDE_VERBOSE ? $z : $z + $a;
     }
 
+    public static function castPeriod(\DatePeriod $p, array $a, Stub $stub, $isNested, $filter)
+    {
+        if (defined('HHVM_VERSION_ID') || \PHP_VERSION_ID < 50605) {
```

Comment:
what happens before 5.6.5?

Reply:
``getStartDate()``, ``getDateInterval()`` and ``getEndDate()`` methods were added in 5.6.5.